lib/Common/Memory/LargeHeapBlock.cpp

```diff
@@ -556,9 +615,28 @@ LargeHeapBlock::Mark(void* objectAddress, MarkContext * markContext)
         return;
     }
 
+    if (header->isPageHeapAlloc)
```

Comment:
removing, using this->InPageHeapMode()

---
In reply to: [60995050](https://github.com/Microsoft/ChakraCore/pull/711#discussion_r60995050) [](ancestors = 60995050)